dali/test/dali_test_single_op.h

```diff
@@ -126,14 +135,21 @@ class DALISingleOpTest : public DALITest {
     if (flags & t_loadJPEGs) {
       LoadJPEGS(images::jpeg_test_images, &jpegs_);
       if (flags & t_decodeJPEGs)
-        DecodeImages(DALI_RGB, jpegs_, &jpeg_decoded_, &jpeg_dims_);
+        DecodeImages(img_type_, jpegs_, &jpeg_decoded_, &jpeg_dims_);
```

Comment:
Good catch! Unfortunately, today we do have only one test
```
class ExecutorTest : public GenericDecoderTest<RGB> {
 protected:
  uint32_t GetImageLoadingFlags() const override {
    return t_loadJPEGs + t_decodeJPEGs;
  }
```
which in fact could pass along this path.